05-html-css/css-grid-intro.md

```diff
@@ -0,0 +1,334 @@
+# Intro to Layout with CSS Grid
+
+## Learning Goals
+
+## History
+
+[Designing on a grid is not a new concept](https://en.wikipedia.org/wiki/Grid_%28graphic_design%29), and in fact grid theory is a well-known practice in many schools of design. However, historically, web designers did not apply these ideas, and CSS did not have available good ways to practice them anyway. It is only until recently that web design has been pushing using a grid for layout.
```

Comment:
I like these "history" sections. It's not just history, it's context and motivation of what problem we're solving and why this is the technique we're teaching them.